config/install/field.field.user.user.field_user_site.yml

```diff
@@ -15,7 +15,7 @@ entity_type: user
 bundle: user
 label: 'Site Restriction'
 description: 'This user will only have editorial access to content belonging to the selected Sites and Sub-sites. This field is optional for roles permitted to bypass site restrictions. See the permissions sub-system for the roles that can bypass.'
```

Comment:
The field description needs to be updated to reflect the changed usage. Older CMS projects like 'content-vic' state that this field can be left empty to allow access to all Sites.

On older CMS projects it may be necessary to update the configuration schemas with the new description. The current "content-vic" project has this description

> "This user will only have editorial access to content belonging to the selected Sites and Sub-sites. **Leave this field empty** to grant the user access to any sites."

See [/config/sync/field.field.user.user.field_user_site.yml](https://github.com/dpc-sdp/content-vic-gov-au/blob/f5c9c54598a26172d8f11cc1ab0a1b48c5e6dd5b/config/sync/field.field.user.user.field_user_site.yml#L18)